2020-09-15 20:48:19,200 INFO    MainThread:137204 [wandb_init.py:_log_setup():307] Logging user logs to wandb/run-20200915_204819-2hlty3p3/logs/debug.log
2020-09-15 20:48:19,200 INFO    MainThread:137204 [wandb_init.py:_log_setup():308] Logging internal logs to wandb/run-20200915_204819-2hlty3p3/logs/debug-internal.log
2020-09-15 20:48:19,200 INFO    MainThread:137204 [wandb_setup.py:_flush():68] setting env: {}
2020-09-15 20:48:19,200 INFO    MainThread:137204 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2020-09-15 20:48:19,201 WARNING MainThread:137204 [wandb_setup.py:_flush():68] Ignoring settings passed to wandb.setup() which has already been configured.
2020-09-15 20:48:19,201 INFO    MainThread:137204 [wandb_setup.py:_flush():68] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f6814218b50>
2020-09-15 20:48:19,201 INFO    MainThread:137204 [wandb_run.py:finish():864] finishing run iit-bhu/emnist/39q30krw
2020-09-15 20:48:19,201 INFO    MainThread:137204 [wandb_init.py:_jupyter_teardown():239] cleaning up jupyter logic
2020-09-15 20:48:19,202 INFO    MainThread:137204 [jupyter.py:save_history():200] not saving jupyter history
2020-09-15 20:48:19,202 INFO    MainThread:137204 [wandb_run.py:_atexit_cleanup():1081] got exitcode: 0
2020-09-15 20:48:19,202 INFO    MainThread:137204 [wandb_run.py:_restore():1053] restore
2020-09-15 20:48:47,922 INFO    MainThread:137204 [wandb_run.py:_wait_for_finish():1185] got exit ret: uuid: "fd52223f41a14a9ea8bc2ad9fd50d0da"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
      media_count: 2
      other_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1505968
      total_bytes: 1505968
    }
  }
}

